### PR TITLE
coucou, first pull request, add new lcg-cp with closeSE

### PR DESCRIPTION
**Pull request by anchenchai**
first pull request try

### CODE REVIEW

**Comment by anchenchai** [comment]
add lcg-lr in LCG.java, add a loop in Gate.java

File: src/Gate.java [hunk 1]
```diff
@@ -7,12 +7,16 @@
...
+
+import java.util.Vector;
+
 import org.simgrid.msg.Host;
 import org.simgrid.msg.Process;
 import org.simgrid.msg.MsgException;
 
 public class Gate extends Job {
 
+	private String[] localOutput = {"/scratch/gate.sh.tar.gz", "/scratch/gate_release.tar.gz", "/scratch/file.zip"};
```
Comment (frs69wq) [inline src/Gate.java]: This array is common to every Gate job. It should be a global variable (and a Vector) in VIPSimulator.java. For instance;
public static Vector<String> gateInputLocalNames = new Vector<String>("/scratch/gate.sh.tar.gz", "/scratch/gate_release.tar.gz", "/scratch/file.zip");

File: src/Gate.java [hunk 2]
```diff
@@ -104,31 +108,30 @@ public void main(String[] args) throws MsgException {
...
-							VIPServer.getDefaultLFC());
-					System.err.println(jobId + "," + getHost().getName() + ","
-							+ transfer_info + ",2");
-
-					transfer_info = LCG.cp(
-							VIPSimulator.gateInputFileNames.get(2),
-							"/scratch/file.zip", VIPServer.getDefaultLFC());
-					System.err.println(jobId + "," + getHost().getName() + ","
-							+ transfer_info + ",2");
+					Vector<SE> replicaLocations = new Vector<SE>();
```
Comment (frs69wq) [inline src/Gate.java]: now need for the "new Vector<SE>()", I think.

File: src/Gate.java [hunk 3]
```diff
@@ -104,31 +108,30 @@ public void main(String[] args) throws MsgException {
...
+					String logicalFileName;
+					
+					for(int i=0;i< VIPSimulator.gateInputFileNames.size();i++){
```
Comment (frs69wq) [inline src/Gate.java]: Replace by (untested)
Iterator iLocalNames = VIPSimulator.gateInputLocalNames.iterator();
for (String LogicalFileName : VIPSimulator.gateInputFileNames){
    localFileName = iLocalNames.next() // that replaces localOuput[i]

Comment (anchenchai) [inline src/Gate.java]: Ok, we could discuss that monday. Because i implemented it in a different way.

**Comment by anchenchai** [comment]
Notification of commit....

**Pull request merged**